src/SprykerEco/Glue/UnzerRestApi/Processor/Mapper/RestCheckoutDataResponseAttributesMapper.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Glue\UnzerRestApi\Processor\Mapper;
+
+use Generated\Shared\Transfer\RestCheckoutDataResponseAttributesTransfer;
+use Generated\Shared\Transfer\RestCheckoutDataTransfer;
+
+class RestCheckoutDataResponseAttributesMapper implements RestCheckoutDataResponseAttributesMapperInterface
```

Comment:
```suggestion
class CheckoutDataResponseMapper implements CheckoutDataResponseMapperInterface
```
Don't forget to update factory method name

Reply:
Resolved in consultation with Ilya Kubanov.